src/networking.rs

```diff
@@ -137,16 +137,23 @@ impl AsResult for Response {
 #[cfg(test)]
 mod tests {
     use super::*;
-
+    use crate::simple_logger;
+    //To run these tests use: 'cargo test -- --ignored'
     #[test]
+    #[ignore] 
     fn get_reports_is_ok() {
+        simple_logger::setup_boxed(LevelFilter::Debug, false);
```

Comment:
I did find 2 options:
* a crate providing this ability, ie [Shiny](https://github.com/farcaller/shiny)
* tests are run in alphabetical order (fact that is implementation detail), so if I put a test named a_0 in lib.rs with logger initialization code, it will be run first and we would have the logs displayed...

Current solution is to initialize the logger in the begining of every test.